extensions/amp-gfycat/0.1/validator-amp-gfycat.protoascii

```diff
@@ -0,0 +1,67 @@
+#
+# Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-gfycat
+  tag_name: "SCRIPT"
```

Comment:
@Gregable I looked at other components code - they have this code in both `tags` sections - for the script tag and for the <amp-component-name> tag. Should I do the same?